src/utils_log.js

```diff
@@ -190,6 +190,7 @@
  */
 
 // variable can't be the same from the namespace above
-import * as Log from 'apify-shared/log';
+import log, { Log, LoggerOptions, LogLevel, Logger, LoggerJson, LoggerText } from '@apify/log';
 
-export default Log;
+export { Log, LoggerOptions, LogLevel, Logger, LoggerJson, LoggerText };
+export default log;
```

Comment:
We created this "hack" to make the docs and types compatible, but now that `@apify/log` is in TS, maybe we don't need this extra file anymore?

Thinking about it more, if we added the JSDoc to `@apify/log` itself, the build of docs probably would not work, so I guess we'll have to live with this solution.

Reply:
yeah i think the script would have to be adjusted to support 3rd party modules. its some homemade thing, right?

will review how the docs work locally tomorrow, i can see some usage of `apify-shared` there too (e.g. in `build_docs.js`)